src/controllers/features/common/typeOfBusinessController.ts

```diff
@@ -41,7 +42,9 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
             const acspData = await getAcspRegistration(session, session.getExtraData(SUBMISSION_ID)!, res.locals.applicationId);
             if (acspData !== undefined) {
                 saveDataInSession(req, USER_DATA, acspData);
-                if (acspData.typeOfBusiness === "UNINCORPORATED" || acspData.typeOfBusiness === "CORPORATE_BODY") {
+                if (defaultBusinessType === "true" && (acspData.typeOfBusiness === "LLP" || acspData.typeOfBusiness === "LC" || acspData.typeOfBusiness === "CORPORATE_BODY")) {
```

Comment:
This seems wrong to me.
when typeOfBusiness is llp etc why are setting it to Sole Trader?

Reply:
When the user clicks on the link to apply as a sole trader from the kick out page, the radio button needs to be checked as a sole trader as the business type, but the user is returning from one of those LLP,LC or CORPORATE_BODY, so the type of business is preset as either of LLP,LC or CORPORATE.

The above is set as sole trader as the user intended to apply as a sole trader, from the AML kick out page, despite starting their journey as one of LC/LLP/CORPORATE